Dockerfile

```diff
@@ -29,9 +30,15 @@ RUN yum -y install php \
     subversion \
     bzip2 \
     php-pecl-json \
+    wget \
     subversion && \
     yum clean all
 
+# REPOFORGE
+RUN wget http://pkgs.repoforge.org/rpmforge-release/rpmforge-release-0.5.3-1.el5.rf.i386.rpm && \
```

Comment:
Because I don't know there is one in EPEL.

I will try to add php-pecl-zip with EPEL repository